rpc/src/rpc.rs

```diff
@@ -1761,8 +1761,8 @@ impl JsonRpcRequestProcessor {
                             bigtable_before = None;
                         }
                         Err(err) => {
-                            warn!("{:?}", err);
-                            return Ok(map_results(results));
```

Comment:
Previously, this code would catch _all_ manner of Bigtable failure, including connection failures, and return whatever results we have so far.